apps/dev/sveltekit/package.json

```diff
@@ -11,8 +11,8 @@
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch"
   },
   "devDependencies": {
-    "@sveltejs/adapter-auto": "next",
-    "@sveltejs/kit": "next",
+    "@sveltejs/adapter-auto": "2.0.0",
+    "@sveltejs/kit": "1.5.2",
```

Comment:
In order to make the sample a bit more stable, I fixed to the most recent versions